kong/db/dao/snis.lua

```diff
@@ -18,6 +18,31 @@ local function invalidate_cache(self, old_entity, err, err_t)
 end
 
 
+local function check_private_key_exists(self, sni)
+  if not sni.certificate or not sni.certificate.id then
+    -- just let validator do it's job
```

Comment:
typo: `it's` -> `its`

Reply:
I always thought `_` had special meaning on its own, until today... Thanks for pointing it out!